gatsby/gatsbysourceplone.rst

```diff
@@ -0,0 +1,62 @@
+gatsby-source-plone
+===================
+
+With the previous sections on source nodes, retrieving data from ``plone.restapi`` and finally using the search traversal method, we have understood how our source-plugin works at base level.
+
+Cheers! That was amazing work.
+
+We have already built this plugin with additional helpful features and functionality to handle all kinds of data, caching and so on in an optimal manner.
+
+So let us try out the `gatsby-source-plone <https://github.com/collective/gatsby-source-plone/>`_ plugin with our `gatsby-starter-plone <https://github.com/collective/gatsby-starter-plone/>`_ to instantly kickstart GatsbyJS development with Plone.
+
+With the GatsbyJS CLI installed, starting up is as simple as:
+
+.. code-block:: console
+
+  gatsby new gatsby-plone-training https://github.com/collective/gatsby-starter-plone
+  
+
+This is will setup a GatsbyJS project in the gatsby-plone-training folder with gatsby-source-plone setup already along with a couple of useful extra features.
+
+To see the starter along with the plugin in action, just run these and navigate to ``localhost:8000``.
```

Comment:
what do you mean with "run?"

gatsby develop command? Write it out before saying "got to localhost:8000".
And it's better to make the url a proper link.